### PR TITLE
Update django compat to 1.8, 1.9, 1.10

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by zebuline** [comment]
Refs #22 #23 #24 #25

**Comment by toopy** [comment]
@k4nar OPM ?

File: django_ticketoffice/api.py [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+    CredentialsError,
+    NoTicketError,
+    TicketExpiredError,
+    TicketUsedError,
+    # decorators
+    invitation_required,
+    stamp_invitation,
+    # models
+    Ticket,
+]
```
Comment (k4nar) [inline django_ticketoffice/api.py]: __all__ expects strings, not classes. It's a bit weird, but I've seen errors when doing that.

Comment (toopy) [inline django_ticketoffice/api.py]: bien vu, thx!

**Pull request merged**